src/libs/routes/authMiddleWare.ts

```diff
@@ -34,7 +34,7 @@ const authMiddleware = (module, permissionType) => async (req, res, next) => {
   console.log('inAuthMiddle', { userData });
   if (!userData) {
     next({
-      err: 'Unauthorized',
+      err: 'Unauthorized(3)',
```

Comment:
key must error and do not extra characters like (1).
Use proper status code or message
see http status code

Reply:
fixed